### PR TITLE
Improve v2v log checking function

### PR DESCRIPTION
**Pull request by xiaodwan**
Add support checking logs when it has multiple lines

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
>>> re.search(ptn, buf, re.S).group(0)
'virt-v2v: warning: mount: mount exited with status 32: mount: \n/sysroot/home: /dev/sda2 already mounted on /sysroot/home. (ignored)'
>>> fd = open('7test.log')
>>> buf7 = fd.read()
>>> re.search(ptn, buf7, re.S).group(0)
'virt-v2v: warning: mount: mount exited with status 32: mount: /dev/sda2 is \nalready mounted or /sysroot/home busy\n       /dev/sda2 is already mounted on /sysroot/home (ignored)'
>>> ptn
'virt-v2v: warning: mount: mount exited with status 32: mount:.*? already mounted .*?\\(ignored\\)'
>>>

**Comment by xiaodwan** [comment]
@chloerh @chunfuwen Please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ def _check_log(pattern_list, expect=True):
             expected = 'expected' if expect else 'not expected'
             logging.info('Searching for %s log: %s' % (expected, pattern))
             compiled_pattern = re.compile(line)
-            search = re.search(compiled_pattern, log)
+            search = re.search(compiled_pattern, log, re.S)
```
Comment (chunfuwen) [inline virttest/utils_v2v.py]: change to re.s -->flags=re.s ?, it looks like more meaningful.

Comment (xiaodwan) [reply virttest/utils_v2v.py]: Done

**Pull request merged**